Examples/src/main/kotlin/com/mapbox/vision/examples/ExternalVideoSourceActivityKt.kt

```diff
@@ -170,6 +176,15 @@ class ExternalVideoSourceActivityKt : BaseActivity() {
                     ImageFormat.RGBA,
                     imageSize
                 )
+                // set CameraParameters them each time they change
```

Comment:
I'd add the comment exactly here ("...that this features still won't work for this particular example...")

Reply:
Well, ok